app/src/main/java/org/gdg/frisbee/android/common/GdgActivity.java

```diff
@@ -45,15 +50,21 @@
 import org.gdg.frisbee.android.utils.PrefUtils;
 import org.gdg.frisbee.android.utils.RecentTasksStyler;
 import org.gdg.frisbee.android.utils.Utils;
+import org.gdg.frisbee.android.view.ColoredSnackBar;
 
 import java.util.List;
 
+import butterknife.Bind;
 import butterknife.ButterKnife;
 import timber.log.Timber;
 
 public abstract class GdgActivity extends TrackableActivity implements
         GoogleApiClient.ConnectionCallbacks, GoogleApiClient.OnConnectionFailedListener {
 
+    @Nullable
+    @Bind(R.id.content_frame)
+    FrameLayout mContentLayout;
```

Comment:
Added this View here in the Base Activity. It is present in most of our views. Below in `showError` method, I show the Snackbar if this is present. I show Toast, if this cannot be found.